Addition of 2 Extra Graph Types

everlybolivar opened the pull request:
In accordance to Issue #61, a graphical view of a stock can now be shown in another two different graph types (RSI and MACD).

everlybolivar committed:
UI Addition - #61
Beginning of adding two other types of graphs.
Modification to the Chart Layout includes the addition of buttons of the two graphs (RSI & MACD) (Task 2)
Slight modification to previous Chart Layout includes better labelling and titles of buttons, for it to be more clear to users

everlybolivar committed:
UI Addition - #61 Beginning of adding two other types of graphs. Modification to the Chart Layout includes the addition of buttons of the two graphs (RSI & MACD) (Task 2) Slight modification to previous Chart Layout includes better labelling and titles of buttons, for it to be more clear to usersUI Addition - #61 Beginning of adding two other types of graphs. Modification to the Chart Layout includes the addition of buttons of the two graphs (RSI & MACD) (Task 2) Slight modification to previous Chart Layout includes better labelling and titles of buttons, for it to be more clear to usersUUI Addition - #61 Beginning of adding two other types of graphs. Modification to the Chart Layout includes the addition of buttons of the two graphs (RSI & MACD) (Task 2) Slight modification to previous Chart Layout includes better labelling and titles of buttons, for it to be more clear to usersI Addition - #61 Beginning of adding two other types of graphs. Modification to the Chart Layout includes the addition of buttons of the two graphs (RSI & MACD) (Task 2) Slight modification to previous Chart Layout includes better labelling and titles of buttons, for it to be more clear to users

everlybolivar committed:
UI Addition - #61 Beginning of adding two other types of graphs. Modification to the Chart Layout includes the addition of buttons of the two graphs (RSI & MACD) (Task 2) Slight modification to previous Chart Layout includes better labelling and titles of buttons, for it to be more clear to users

everlybolivar committed:
Functionality Implemented - #61
Button functionality for both RSI and MACD buttons have been implemented (Task 3). When clicked, a graph corresponding to the button will be presented. Functionality was implemented in the ChartActivity file, while logic for pulling data for the graphs was added to the GlobalWidgetData file - as seen in all previous Chart implementations.
Minor changes that also occured was the renaming of the buttons to conform to the team's coding style.

everlybolivar committed:
Fixed minor typo in pulling MACD function values within ChartActivity.

everlybolivar committed:
Refactoring to try and fix CodeClimate errors.

everlybolivar committed:
Refactor in attempt to fix CodeClimate errors V2

Nick9500 commented on src/main/java/nitezh/ministock/activities/ChartActivity.java:
```diff
@@ -90,15 +97,55 @@ public void onClick(View v) {
+
+            new ImageSnatcher((ImageView) findViewById(R.id.chart_img)).execute(GlobalWidgetData.constructImageUrl(graphPrices));
+        }
+        /* else if (interval == 5) {
```
Remove comments please

everlybolivar replied:
Done

everlybolivar committed:
Fixed requested changes from reviewer.
Also added the re-set of the interval (chosen graphs) to the default (Weekly Stock prices) once user exits the graph.
